internal/transformer/checkNaN.go

```diff
@@ -9,19 +9,13 @@ package transformer
 import (
 	"math"
 
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
 	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2"
 
-	dsModels "github.com/edgexfoundry/device-sdk-go/v2/pkg/models"
+	"github.com/edgexfoundry/device-sdk-go/v2/pkg/models"
 )
 
-// NaNError is used to throw the NaN error for the floating-point value
-type NaNError struct{}
-
-func (e NaNError) Error() string {
-	return "not a valid float value NaN"
-}
-
-func isNaN(cv *dsModels.CommandValue) (bool, error) {
+func isNaN(cv *models.CommandValue) (bool, errors.EdgeX) {
```

Comment:
use errors.Wrap to wrap the err before returning it

Reply:
do we need to wrap the error if all the returning error is `errors.Edgex` type ?